extension.js

```diff
@@ -60,7 +35,7 @@ function gremlinsFromConfig(context) {
       hexCodePoint,
       decorationType: vscode.window.createTextEditorDecorationType(
         decorationType,
-      ),
+      )
```

Comment:
We use Prettier to format our code, and like the trailing commas:
https://github.com/nhoizey/vscode-gremlins/blob/master/package.json#L60-L64
https://prettier.io/docs/en/options.html#trailing-commas

Didn't your IDE use Prettier when saving your edits?